### ISSUE TITLE
Accessibility - Poor Color Contrast of Footer Icons

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#3]
There is poor color contrast between the gray icons (#A0A1A1 ) in the footer and the white background ( #ffffff ). The color contrast ratio is 2.39:1 and needs to be 4.5:1 or higher to pass. There is also a color contrast issues between the line that denotes the start of the footer too.

**Comment by aanthonyDOL** [comment]
What would be the suggested approach to making this 508 Compliant, 1) Make the icons Darker

**Comment by jfc3-dol** [comment]
Yes, making them darker with a color contrast ratio of 4.5:1 or higher. To get that you could use #767676 to squeak by and pass or make it more a charcoal gray of #3d3d3d. Think you should look at what other colors you have already used to determine which one to sue but they both will pass.

**Comment by jfc3-dol** [comment]
The color contrast ratio for the footer icons ( #A1A1A1 ) against the white background ( #ffffff ) is 2.58:1 and needs to be 4,5:1 or higher to pass.

If icons are using #767676 or something darker such as #3d3d3d, then I'm assuming the application is using opacity on the icons which are causing it not to have enough color contrast.

You can use online tool such as Jonathon Snook's Colour Contrast Checker ( https://snook.ca/technical/colour_contrast/colour.html ) to enter the icon color and the background color to see if the foreground and background pass by looking at the "Colour Contrast Ratio" is 4.5:1 or higher and the two "WCAG 2 AA Compliant" items below it say "YES".
